queue/listener_test.go

```diff
@@ -0,0 +1,195 @@
+package queue_test
```

Comment:
Even if the implementation is SQS free, it's good to have the test using it (even if only as an implementation example 😂 )